lesson5_tkgiang/quydoitien.php

```diff
@@ -0,0 +1,261 @@
+<?php
+	class Amount 
+	{
+		public $amount;
+    	public function __construct($amount) {
+        	$this->amount = $amount;
+    	}
+	}
+	class UsdTo extends Amount 
+	{
+		function usdToVnd() {
+			return $this->amount * 22943;
+		}
+		function usdToGbp() {
+			return $this->amount * 0.72;
+		}
+		function usdToEur() {
+			return $this->amount * 0.85;
+		}
+		function usdToCny() {
+			return $this->amount * 6.48;
+		}
+		function usdToUsd() {
+			return $this->amount;
+		}
+	}
+	class GbpTo extends Amount 
+	{
+		function gbpToVnd() {
+			return $this->amount * 31816;
+		}
+		function gbpToGbp() {
+			return $this->amount;
+		}
+		function gbpToEur() {
+			return $this->amount * 1.18;
+		}
+		function gbpToCny() {
+			return $this->amount * 9;
+		}
+		function gbpToUsd() {
+			return $this->amount * 1.39;
+		}
+	}
+	class EurTo extends Amount 
+	{
+		function eurToVnd() {
+			return $this->amount * 27069;
+		}
+		function eurToGbp() {
+			return $this->amount * 0.85;
+		}
+		function eurToEur() {
+			return $this->amount;
+		}
+		function eurToCny() {
+			return $this->amount * 7.63;
+		}
+		function eurToUsd() {
+			return $this->amount * 1.18;
+		}
+	}
+	class CnyTo extends Amount 
+	{
+		function cnyToVnd() {
+			return $this->amount * 3536;
+		}
+		function cnyToGbp() {
+			return $this->amount * 0.11;
+		}
+		function cnyToEur() {
+			return $this->amount * 0.13;
+		}
+		function cnyToCny() {
+			return $this->amount;
+		}
+		function cnyToUsd() {
+			return $this->amount * 0.15;
+		}
+	}
+	class VndTo extends Amount 
+	{
+		function vndToVnd() {
+			return $this->amount;
+		}
+		function vndToGbp() {
+			return $this->amount * 0.000031;
+		}
+		function vndToEur() {
+			return $this->amount * 0.000037;
+		}
+		function vndToCny() {
+			return $this->amount * 0.00028;
+		}
+		function vndToUsd() {
+			return $this->amount * 0.000043;
+		}
+	}
+	$default = '';
+    $convertNumber = '';
+    if(isset($_POST['input'])) {
```

Comment:
```suggestion
    if (isset($_POST['input'])) {
```
check thêm isset `from`, `to` nữa chứ e nhỉ